test/board_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns board-test
+  (:require [clojure.test :refer [deftest testing is]]
+            [board :refer :all]))
+
+(deftest board-test
+  (testing "creates a fresh board"
+    (is (= [1 2 3 4 5 6 7 8 9] (create-board))))
+
+  (testing "updates board when given a location and mark"
+    (is (= ["X" 2 3 4 5 6 7 8 9] (update-board [1 2 3 4 5 6 7 8 9] 0 "X")))))
+
+;;  (testing "formats board into presentable string when given a board"
```

Comment:
Is there a reason why this test if commented out?

Reply:
I was debating whether it is the board's job to format the board or if it is the display's job.